embrace-gradle-plugin/src/main/java/io/embrace/android/gradle/plugin/api/EmbraceBytecodeInstrumentation.kt

```diff
@@ -41,6 +41,16 @@ abstract class EmbraceBytecodeInstrumentation @Inject internal constructor(objec
      */
     val autoSdkInitializationEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java)
 
+    /**
+     * Whether Embrace should automatically instrument application initialization timing start. Defaults to true.
+     */
+    val applicationInitTimeStartEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java).convention(true)
+
+    /**
+     * Whether Embrace should automatically instrument application initialization timing end. Defaults to true.
+     */
+    val applicationInitTimeEndEnabled: Property<Boolean> = objectFactory.property(Boolean::class.java).convention(true)
```

Comment:
IMO these should be one flag, `applicationInitTimeEnabled`. I can't think of a scenario where folks would only want to disable one.

Reply:
you're right, I think there's something that fails the build if you don't have flags for a certain bytecode instrumentation, so I added a flag to make that pass, but I forgot to remove it 🤦‍♂️